TACC/Core-Portal
# task/FP-644 - Show an error message if user is missing default storage system host allocation

## Overview: ##

Show an error message if user is missing default storage system host allocation

## PR Status: ##


## Related Jira tickets: ##

* [FP-644](https://jira.tacc.utexas.edu/browse/FP-644)

## Summary of Changes: ##

- SystemsListView returns defaultHost (hostname of default system)
- Client checks allocations.host for presence of defaultHost

## Testing Steps: ##
1. Use a test account with no allocations. Load a SLURM app - it should display an error that you need an allocation on frontera.tacc.utexas.edu

## UI Photos:

![image](https://user-images.githubusercontent.com/17181582/90562378-aedfb900-e167-11ea-9ae7-ff973cb8c1c2.png)


## Notes: ##

I'd like to merge tests from https://github.com/TACC/Frontera-Portal/pull/170 before adding tests here.
